src/WalkingModule/app/robots/ergoCubGazeboV1/dcm_walking/common/navigation.ini

```diff
@@ -0,0 +1,10 @@
+# Navigation trigger delay in seconds. Time to wait before sending the replanning command after exiting the double support stance.
+navigationReplanningDelay   0.9
+# Loop rate for the thread computing the navigation trigger
+navigationTriggerLoopRate   100
```

Comment:
What is the unit of measurement of this value?

Reply:
The first is seconds, the second is hertz

Reply:
> the second is hertz

Can you specify it either in the name or in a comment?

Reply:
Done